### ISSUE TITLE
FEAT: add filter/sorting system

### ISSUE THREAD

**Issue by BachErik** [BachErik/killedby#24]
**Is your feature request related to a problem? Please describe.**
The header does not look good on mobile phones if you have added too many companies. At first I thought of a simple expandable menu. But I would like to be able to search by year and by project type. What would be good to find a project again would be a searchbar with a search function to which the filters can also be applied.

**Describe the solution you'd like**
To summarise briefly: I would like to have a header on which mam can achieve a kind of filter system on all devices and has a search function. You should be able to filter for:
- years
- Project types
- company
